frontend/src/bundles/studio/pages/studio.tsx

```diff
@@ -1,13 +1,40 @@
+import { minutesToMilliseconds } from 'date-fns';
 import {
+    type ItemDefinition,
+    type Range,
+    type RowDefinition,
+} from 'dnd-timeline';
+
+import {
+    Box,
     Button,
     Header,
     Icon,
     IconButton,
+    VStack,
 } from '~/bundles/common/components/components.js';
 import { IconName } from '~/bundles/common/icons/icons.js';
+import {
+    generateMockItems,
+    generateMockRows,
+} from '~/bundles/studio/mocks/mock.helper.js';
 
+import styles from '../components/timeline/styles.module.css';
```

Comment:
maybe its better to create it for studio instead of timeline

because you use it in studio page component not in timeline directly

Reply:
I thought about this, ok, I'll move it to studio module css and maybe it would have more styles not only for the timeline.

Reply:
Done)